openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```diff
@@ -585,42 +586,42 @@
     "no-inherited-roles-found": "No inherited roles found",
-    "no-kpi-available-add-new-one": "No KPI's are available, add one by clicking Add KPI button.",
+    "no-kpi-available-add-new-one": "No KPIs are available. Click on the Add KPI button to add one.",
     "no-kpi-found": "No KPI found with name {{name}}",
-    "no-notification-found": "No notifications Found",
+    "no-notification-found": "No notifications found",
     "no-permission-for-action": "You do not have the necessary permissions to perform this action.",
     "no-permission-to-view": "You do not have the necessary permissions to view this data.",
     "no-policy-found-for": "No policy found for",
-    "no-profiler-enabled-summary-message": "Profiler is not enabled for the table.",
-    "no-profiler-message": "Data Profiler is an optional configuration in Ingestion. Please enable the data profiler by following the documentation",
+    "no-profiler-enabled-summary-message": "Profiler is not enabled for this table.",
+    "no-profiler-message": "Data Profiler is an optional configuration in Ingestion. Please enable the data profiler by following the documentation.",
     "no-roles-assigned": "No roles assigned",
     "no-roles-found-for": "No roles found for",
     "no-rule-found": "No rule found",
     "no-schema-data-available": " No schema data available",
-    "no-selected-dbt": "No source selected for DBT Configuration.",
+    "no-selected-dbt": "No source selected for dbt Configuration.",
     "no-service-connection-details-message": "{{serviceName}} doesn't have connection details filled in. Please add the details before scheduling an ingestion job.",
     "no-team-found": "No team found.",
     "no-terms-found": "No terms found",
     "no-terms-found-for-search-text": "No terms found for {{searchText}}",
     "no-user-available": "No user available",
     "no-users": "There are no users {{text}}",
-    "om-description": "centralized metadata store, discover, collaborate and get your data right",
+    "om-description": "Centralized metadata store, to Discover, collaborate and get your data right.",
     "permanently-delete-metadata": "Permanently deleting this {{entityName}} will remove its metadata from OpenMetadata permanently.",
     "permanently-delete-metadata-and-dependents": "Permanently deleting this {{entityName}} will remove its metadata, as well as the metadata of {{dependents}} from OpenMetadata permanently.",
-    "pipeline-trigger-success-message": "Pipeline triggered successfully",
+    "pipeline-trigger-success-message": "Pipeline triggered successfully!",
     "profile-sample-percentage-message": "Set the Profiler value as percentage",
     "profile-sample-row-count-message": " Set the Profiler value as row count",
     "profiler-timeout-seconds-message": "Optional number setting the timeout in seconds for the profiler. If the timeout is reached the profiler will wait for any pending queries to terminated its execution.",
```

Comment:
Unclear message.... Is it supposed to be:

For the Profiler, add the duration in seconds for timeout (optional). The profiler will wait for any pending queries to be executed, or will terminate them once it reaches the timeout.